server/src/test/java/com/google/coffeehouse/StorageHandlerTest.java

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.coffeehouse;
+
+import static com.google.cloud.spanner.TransactionRunner.TransactionCallable;
+
+import com.google.cloud.spanner.Database;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Instance;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.KeySet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+import com.google.cloud.spanner.TransactionContext;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+* The StorageHandlerTest class encompasses functions to set up the spanner service
+* and database client, insert data for each specific test, and then deletes the data in tearDown().
+*/
+@RunWith(JUnit4.class)
+public class StorageHandlerTest {
+
+  static DatabaseClient dbClient;
+  private static final String INSTANCE_ID = "coffeehouse-instance-test";
+  private static final String DATABASE_ID = "coffeehouse-db-test";
+
+  @Before
+  public void setUp() {
+    // Create spanner service and database client
+    SpannerOptions options = SpannerOptions.newBuilder().build();
+    Spanner spanner = options.getService();
+    DatabaseId db = DatabaseId.of(options.getProjectId(), INSTANCE_ID, DATABASE_ID);
+    dbClient = spanner.getDatabaseClient(db);
+
+    // Delete all the data while keeping the existing database
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(Mutation.delete("Persons", KeySet.all()));
+    dbClient.write(mutations);
+  }
+
+  public void insertPersonWithPronouns() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertOrUpdateBuilder("Persons")
+        .set("userId")
+        .to("pronouns")
```

Comment:
Nit: For clarity, consider changing this to "hasPronouns" or something similar. It took me a moment to parse this with the duplicated "pronouns."